### ISSUE TITLE
Validation error when creating new translation

### ISSUE THREAD

**Issue by PLawitschka** [silverstripe/silverstripe-translatable#179]
Hey everyone,

i'm getting an Validation error when I'm trying to create a new translation saying: 
```
"Please select a value within the list provided. en_US is not a valid option"
```
 It worked for the first translated page but after that I always get that error. I think it has something to do with the validation of the GroupedDropdownField (parent of LanguageDropDownField). It seems to not validate nested arrays since it hasn't an own validate function but uses that one of its parent.

But I may be missing something...

**Comment by chillu** [comment]
Have you modified `i18n::$common_locales` or `Translatable::$allowed_locales`?

**Comment by chillu** [comment]
Also, can you reproduce this on a default 3.1 installation on an empty database?

**Comment by chillu** [comment]
Actually, this is a regression from Nov 2014 in the 3.2 pre-release branch of core, see 
https://github.com/silverstripe/silverstripe-framework/commit/41ea83b3374d026da9575f5256a02bdcfbb82059

PR sent in https://github.com/silverstripe/silverstripe-framework/pull/3921

**Comment by chillu** [comment]
This has been merged into framework and is part of the 3.1.12 core release.
